src/system/components/handler.clj

```diff
@@ -71,4 +80,11 @@
                       (component/using [:endpoint-a :endpoint-b :middleware]))
          :jetty (-> (new-web-server port)
                     (component/using [:handler])))"
-  ([] (->Handler)))
+  ([] (->Handler :compojure))
```

Comment:
We are slowly moving from positional arguments to keyword arguments in component constructors. 
Please refer to `konserve` component to see how it's done now.